tests/bluetooth/bsim_bt/bsim_test_eatt_notif/src/client_test.c

```diff
@@ -153,6 +153,7 @@ static void gatt_discover(void)
 	discover_params.start_handle = BT_ATT_FIRST_ATTRIBUTE_HANDLE;
 	discover_params.end_handle = BT_ATT_LAST_ATTRIBUTE_HANDLE;
 	discover_params.type = BT_GATT_DISCOVER_PRIMARY;
+	discover_params.chan_opt = BT_ATT_CHAN_ANY;
```

Comment:
I would actually love to see a test to cover the two options to verify is they work as intended.

Reply:
I am working on a test now.

Reply:
I might be missing something; how is the test actually using the different configurations? It still, to me, look like the tests are still just using `CHAN_OPT_NONE`

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/45332/files#diff-09e781ebc5a36992b15e9b3587153a67fe2cfe524c8c0571a69e80ad207ec85bR225. The latest commit adds variations to the existing notify bsim test using different `CHAN_OPT`s.